libtransmission/mime-types.h

```diff
@@ -1264,6 +1304,7 @@ inline auto constexpr mime_type_suffixes = std::array<mime_type_suffix, 1250>{
       { "z7", "application/x-zmachine" },
       { "z8", "application/x-zmachine" },
       { "zaz", "application/vnd.zzazz.deck+xml" },
+      { "zip", "application/x-zip-compressed" },
       { "zip", "application/zip" },
```

Comment:
Unrelated to this PR but I don't think we want duplicates here…

Reply:
#7590 should fix this.